### DISCUSSION TITLE
how will we know if a car feature or any type is local or global

### DISCUSSION THREAD

**Discussion by infinity-stack** [Deadlineem/Extras-Addon-for-YimMenu#130] [Q&A]
is there a way where we can know what we do people can see? and another question are you guys able to do something like construction (lua script from stand)

**Comment by Deadlineem** [comment]
The things people can and cant see are based on if they are networked or not.  In particle effects cases, you only see some of them regardless if they are networked or not.  For things like the object spawner, they can see the objects you are previewing/spawn in but for tokyo drift, the nos effect and purge cant be seen by others.

Fireworks can be seen by others because it uses a networked ptfx from the firework launcher  Basically any online based effects can be seen by others in a nutshell.

**Reply by infinity-stack** [reply]
what about that one lua script from stand called construction like where you can make stuff with existing addons or something https://github.com/hexarobi/stand-lua-constructor

**Reply by Deadlineem** [reply]
> what about that one lua script from stand called construction like where you can make stuff with existing addons or something https://github.com/hexarobi/stand-lua-constructor

I have hexarobi in my channel for extras i can ask him if he would do a variation for Yim.

**Reply by infinity-stack** [reply]
oh really, when?

**Reply by Deadlineem** [reply]
> what about that one lua script from stand called construction like where you can make stuff with existing addons or something https://github.com/hexarobi/stand-lua-constructor

![Screenshot_175](https://github.com/user-attachments/assets/2c71dd27-0ed6-40ff-be61-d73a0d2ae93f)

**Reply by infinity-stack** [reply]
that's nice man

**Reply by infinity-stack** [reply]
what discord is that?

**Reply by infinity-stack** [reply]
did they say anything?

**Reply by hexarobi** [reply]
Happy to help answer Q's if I can, but I'm prob not willing to take on porting the whole script to Yim. FWIW most of the constructor_lib and convertors file should be vanilla Lua (maybe need to add a pairs() wrapper around for loops?) the hard part will be the menu stuff.

**Reply by infinity-stack** [reply]
Ok
